### PR TITLE
hal_nxp: netc

### PR DESCRIPTION
**Pull request by yangbolu1991**

### CODE REVIEW

File: mcux/mcux-sdk/drivers/netc/socs/imx943/fsl_netc_soc.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```
Comment (JiafeiPan) [inline mcux/mcux-sdk/drivers/netc/socs/imx943/fsl_netc_soc.c:4]: remove all code for imx943?

**Review by JiafeiPan — CHANGES_REQUESTED** [review]

File: mcux/mcux-sdk/drivers/netc/socs/imx943/fsl_netc_soc.c (hunk 1 above)
Comment (yangbolu1991) [reply mcux/mcux-sdk/drivers/netc/socs/imx943/fsl_netc_soc.c]: > remove all code for imx943?

Okay. Updated the pull request and removed.
Thanks.

**Comment by yangbolu1991** [comment]
Any other comments?
Thanks.

**Comment by yangbolu1991** [comment]
Will re-send patches together with i.MX95 platform netc support in the future.
Thanks.

**Pull request closed without merging**